.gitignore

```diff
@@ -2,7 +2,6 @@ node_modules
 grassroots-backend/dist
 grassroots-backend/src/metadata.ts
 grassroots-frontend/dist
-grassroots-frontend/src/**/*.js
```

Comment:
These files are annoying, and this makes it easy to just git clean them away.

Reply:
They currently should only be put in the dist folder. I just accidentally do a raw tsc forgetting --noEmit rather regularly...